test/lit/passes/inlining_splitting.wast

```diff
@@ -203,18 +201,8 @@
     (call $br-to-toplevel (i32.const 2))
   )
 
-  ;; CHECK:      (func $nondefaultable-param (param $x i32) (param $y (rtt $struct))
-  ;; CHECK-NEXT:  (if
-  ;; CHECK-NEXT:   (local.get $x)
-  ;; CHECK-NEXT:   (return)
-  ;; CHECK-NEXT:  )
-  ;; CHECK-NEXT:  (loop $l
-  ;; CHECK-NEXT:   (call $import)
-  ;; CHECK-NEXT:   (br $l)
-  ;; CHECK-NEXT:  )
-  ;; CHECK-NEXT: )
-  (func $nondefaultable-param (param $x i32) (param $y (rtt $struct))
-    ;; The RTT param here prevents us from even being inlined, even with
+  (func $nondefaultable-param (param $x i32) (param $y (ref $struct))
+    ;; The non-nullable param here prevents us from even being inlined, even with
```

Comment:
Looks like it is inlined just below... which is ok if we fix up the type to be nullable, which we do. So just the comment should be fixed here. Maybe check if there is already a test for non-nullable locals though in this file, this might be deletable.

Reply:
Good catch. It looks like this gets inlined to use a `(ref null $struct)` local, but I think we could even use a non-nullable `(ref $struct)` local once we handle those well. Does that need to be tracked separately or will it be handled by your work?

Reply:
Yes, that will work automatically once we have 1a support. We'll see that the local is in proper 1a position and keep it non-nullable.